src/adapters/allspark/execution/src/sdk/lib.ts

```diff
@@ -0,0 +1,27 @@
+import { UserTxData } from "./types";
+
+export async function getUserTXAtBlock(startBlock: number, endBlock: number) {
+  const response = await fetch('https://api.allspark.finance/zklink-mantissa-api/getNovaPoint', {
+      method: 'POST',
+      body: JSON.stringify({ startBlock: startBlock, endBlock: endBlock }),
+      headers: { 'Content-Type': 'application/json' },
+    });
+
+  const data = await response.json();
+  const userTxData: UserTxData[] = data.data.map((item: any) => {
+    return {
+      timestamp: item.timestamp,
+      userAddress: item.userAddress,
+      contractAddress: item.contractAddress,
+      tokenAddress: item.tokenAddress,
+      decimals: item.decimals,
+      price: item.price,
```

Comment:
you can omit the price and remove this field

Reply:
solved